data/tools/unit_tree/helpers.py

```diff
@@ -40,11 +26,16 @@ def __init__(self, wesnoth_exe, userdir, datadir, magick_exe="convert"):
         self.images_by_ipath = {}
         self.binary_paths_per_addon = {}
         self.datadir = datadir
-        if not self.datadir:
-            self.datadir = get_datadir(wesnoth_exe)
         self.userdir = userdir
-        if not self.userdir:
-            self.userdir = get_userdir(wesnoth_exe)
+        self.hide_paths = [
+            os.path.join(self.userdir, "data", "add-ons"),
+            os.path.join(self.userdir, "data"),
+            os.path.join(self.userdir),
+            os.path.join(self.datadir, "data", "add-ons"),
+            os.path.join(self.datadir, "data"),
+            os.path.join(self.datadir)
+        ]
+>>>>>>> 8c702a0eda1 (Fixes)
```

Comment:
Since the commits look fine it'd be good to fix this so we don't have to squash merge.